src/globus_cli/commands/flows/create.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import annotations
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import JSONStringOrFile, ParsedJSONData, command
+from globus_cli.termio import Field, TextMode, display, formatters
+from globus_cli.types import JsonValue
+
+ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+
+
+@command("create", short_help="Create a flow")
+@click.argument(
+    "title",
+    type=str,
+)
+@click.argument(
+    "definition",
+    type=JSONStringOrFile(),
+    metavar="DEFINITION",
+)
+@click.option(
+    "--input-schema",
+    "input_schema",
+    type=JSONStringOrFile(),
+    help="""
+        The JSON input schema that governs the parameters
+        used to start the flow.
+
+        The input document may be specified inline, or it may be a path to a JSON file.
+
+        Example: Inline JSON:
+
+        \b
+            --input-schema '{"properties": {"src": {"type": "string"}}}'
+
+        Example: Path to JSON file:
+
+        \b
+            --input-schema schema.json
+
+        If unspecified, the default is an empty JSON object ('{}').
+    """,
+)
+@click.option(
+    "--subtitle",
+    type=str,
+    help="A concise summary of the flow's purpose.",
+)
+@click.option(
+    "--description",
+    type=str,
+    help="A detailed description of the flow's purpose.",
+)
+@click.option(
+    "--administrator",
+    "administrators",
+    type=str,
+    multiple=True,
+    help="""
+        A principal that may perform administrative operations
+        on the flow (e.g., update, delete).
+
+        This option can be specified multiple times
+        to create a list of flow administrators.
+    """,
+)
+@click.option(
+    "--starter",
+    "starters",
+    type=str,
+    multiple=True,
+    help="""
+        A principal that may run the flow.
+
+        Use "all_authenticated_users" to allow any authenticated user to run the flow.
```

Comment:
What would you think about "start a new run of the flow"? I am a little hesitant to remove the word "run" only because it may make the meaning a little ambiguous (I do like the separation of using "start" as a verb and "run" as a noun so that we're not "running runs"—but while it seems clear to me that "starting" a flow creates a "run", I don't think that's necessarily obvious to a new user to the service).

Reply:
> "start a new run of the flow"

Nice. I like it!